Shipr-Helm/templates/shipr-frontend-deployment.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: shipr-frontend
```

Comment:
Do you think there would be any problems we could run into if we keep this as is?

Reply:
I don't think so there is any issue with the name of the Deployment unless there is any other frontend service that has conflict with this deployment name.

Reply:
Helm charts are designed to be reusable and deployable in various environments. Hardcoding resource names can lead to naming collisions when the same chart is deployed multiple times in the same Kubernetes cluster or namespace. If I deploy this helm chart using two different release names, because of this hard-coded name, the release will fail.

Reply:
Used from the values file